### ISSUE TITLE
Accessibility issue jaws

### ISSUE THREAD

**Issue by nagenderRawat** [indrimuska/angular-selector#83]
Whenever i'am trying to change option through keystroke in dropdown(using up arrow and down arrow) jaws(screen reader) does not read the focused option.
